pkg/appconsts/v3/app_consts.go

```diff
@@ -4,6 +4,7 @@ import "time"
 
 const (
 	Version              uint64 = 3
+	NextVersion          uint64 = 4
```

Comment:
Why does this need to exist? Can't you just add a v4 package with the v4 constants

Reply:
This is only useful for the v3 to v4 upgrade. v3 doesn't need to know anything else about v4.
Felt cleaner than creating a whole new package or do v3.Version+1

Reply:
Maybe this PR didn't create a v4 package because this was backported to v3.x and could be confusing to introduce a v4 package on v3.x 🤷‍♂️ 

IMO nothing prevents us from doing that though